app/js/actions/createLabOrder.js

```diff
@@ -0,0 +1,41 @@
+import 'babel-polyfill';
+import randomString, { generate } from 'randomstring';
+
+import loading from './loading';
+import axiosInstance from '../config';
+
+import {
+  CREATE_LAB_ORDER,
+  CREATE_LAB_ORDER_SUCCESS,
+  CREATE_LAB_ORDER_FAILURE,
+} from './actionTypes';
+
+const createLabOrderFailure = error => ({
+  type: CREATE_LAB_ORDER_FAILURE,
+  error,
+});
+
+const createLabOrderSuccess = data => ({
+  type: CREATE_LAB_ORDER_SUCCESS,
+  data,
+});
+
+const createLabOrder = labOrderData => async (dispatch) => {
+  try {
+    dispatch(loading(CREATE_LAB_ORDER, true));
+    const requestPayload = {
+      name: randomString.generate(10),
+      description: JSON.stringify(labOrderData),
+      javaClassName: 'org.openmrs.Laborder',
+    };
+    const response = await axiosInstance.post('/ordertype', requestPayload);
```

Comment:
I see you are saving the Lab Orders into the `orderType` endpoint. I am not sure this a good idea and that is because:

- The main purpose of this endpoint is to store the order types and not the orders themselves. If there is a functionality that depends on the response from this endpoint, that functionality might begin to break.
- What happens when we need to fetch the Lab Orders for display, even though the lab orders are saved there, we would be fetching the order-types and not really the Lab Orders, consequently, displaying the lab orders would be very difficult.

I am aware that you are having a blocker on saving the Lab Orders into the `/order` endpoint as you have posted [here](https://talk.openmrs.org/t/issue-making-a-post-request-to-create-a-test-order-order-type/18908), I suggest you work toward fixing the blocker as opposed to using this approach.

Reply:
the bug is coming from the API itself, and this is just an alternative until the issue with the order resource endpoint is rectified.

Reply:
Please check this: https://talk.openmrs.org/t/issue-making-a-post-request-to-create-a-test-order-order-type/18908/7?u=tunmi
Mark Goodrich is asking you to create a ticket on REST WS module to resolve the bug you are facing. He also commented on the approach on saving the Order through the `orderType` endpoint not being a suitable one.
Please follow up with the bug fix.